index.js

```diff
@@ -33,9 +33,11 @@ const TALKING_POINT_TARGET_CHANNEL_ID = process.env.TALKING_POINT_CHANNEL_ID;
 const GENERAL_CHANNEL_ID = process.env.GENERAL_CHANNEL_ID;
 const USERNAME_REGEX = /<@(\d+)>/g;
 const TODO_REGEX = /TODO/;
+const TALKING_POINT_REGEX = /(Talking point|TP)/;
 const TIME_IN_DAYS = 1000 * 60 * 60 * 24;
 const CHAR_LENGTH = 500;
 const deleteTodo = [false, false];
+let talkingPointCount = 0;
```

Comment:
Is there a way we could have it pick up the talking point count even after the bot restarts? One way would be to check the latest message in the Talking Points channel, read its title, and get the count from that

Reply:
That's definitely doable